babel.config.js

```diff
@@ -39,36 +40,26 @@ module.exports = function(api) {
 					'@babel/react',
 				],
 			},
-			documentation: {
-				plugins: [['@babel/plugin-proposal-object-rest-spread'], 'lodash'],
-				presets: [
-					[
-						'@babel/env',
-						{
-							targets: {
-								browsers: ['last 2 versions'],
-							},
-						},
-					],
-					'@babel/react',
-				],
-			},
 			production: {
 				plugins: [
 					['@babel/plugin-proposal-object-rest-spread'],
-					[
-						'babel-plugin-transform-require-ignore',
-						{
-							extensions: ['.less', '.css'],
-						},
-					],
-					[
-						'react-peek/babel',
-						{
-							minifyStatics: true,
-						},
-					],
 					'lodash',
+					...(isDocsBuild
```

Comment:
kind of hacky, but I wasn't able to get anything else working.  If we leave the ignore/peek plugins in docs lose descriptions and css.  If we remove the plugins the library components jump from 4.5mb to 4.8mb.  Manually injecting the `documentation` NODE_ENV overrides some of babel's `mode: 'production'` settings and we get left with stuff like development React

Some of the docs I was looking at: 
https://babeljs.io/docs/en/config-files
https://babeljs.io/docs/en/options